install/docker/console-db-es-ls-kibana/docker-compose.yml

```diff
@@ -0,0 +1,127 @@
+version: '3.2'
+
+services:
+  db:
+    restart: always
+    image: mariadb:10.3
+    container_name: cws-db
+    ports:
+      - "3306:3306"
+    command: mysqld --max-connections=2000 --transaction-isolation=READ-COMMITTED
+    environment:
+      - MYSQL_DATABASE=cws
+      - MYSQL_ROOT_PASSWORD=test
+      - TZ=America/Los_Angeles
+    healthcheck:
+      test: "/usr/bin/mysql --user=root --password=test --execute \"SHOW DATABASES;\""
```

Comment:
DB username and password exposed... Are these security risks?

Reply:
no